core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/utils/ReplicatedTableProviderImpl.java

```diff
@@ -52,14 +61,66 @@ public boolean accept(String keyspace)
             return false;
         }
 
-        Set<TokenRange> tokenRanges = myMetadata.getTokenRanges(keyspace, myLocalhost);
+        KeyspaceMetadata keyspaceMetadata = myMetadata.getKeyspace(keyspace);
 
-        if (tokenRanges.isEmpty())
+        if (keyspaceMetadata != null)
         {
+            Map<String, String> replication = keyspaceMetadata.getReplication();
+            String replicationClass = replication.get(STRATEGY_CLASS);
+
+            switch(replicationClass)
+            {
+                case SIMPLE_STRATEGY:
```

Comment:
Will you always get class name _including_ packages here?

Reply:
The class name stored in system_schema.keyspaces where it uses Class#getName() to retrieve the name. So I believe this should always be the case.